google/cloud/bigtable/filters_test.cc

```diff
@@ -288,13 +288,13 @@ TEST(FiltersTest, Sink) {
   ASSERT_TRUE(proto.sink());
 }
 
-/// @test Verify that `bigtable::Filter::as_proto_move` works as expected.
+/// @test Verify that `bigtable::Filter::as_proto` works as expected.
 TEST(FiltersTest, MoveProto) {
   using F = bigtable::Filter;
   auto filter = F::Chain(F::FamilyRegex("fam"), F::ColumnRegex("col"),
                          F::CellsRowOffset(2), F::Latest(1));
   auto proto_copy = filter.as_proto();
-  auto proto_move = filter.as_proto_move();
+  auto proto_move = std::move(filter).as_proto();
```

Comment:
Testing moves is hard and typically not worth the effort. But I wonder if it's worth adding a simple static assert like:

```cc
static_assert(std::is_rvalue_reference<decltype(std::move(filter).as_proto())>::value, "");
```

(Note: the above static_assertion would fail if `filter` were const... but I think that's probably fine.)

Reply:
Done.